app/assets/javascripts/backbone/apps/timesheet/entries/time_entry_list_item.hamlc

```diff
@@ -8,7 +8,7 @@
 - if @task
   .task-container
     %span.description-text
-      %a{href: @task, target: '_blank'}= _.last(@task.split('/'))
+      %a{href: @task, target: '_blank'}= _.last(@task.split(/([.;=?/,]+)/g))
```

Comment:
Maybe let's move this entirely into json response and add some test cases to make sure we handle all cases from most popular issue tracking tools like Jira, Pivotal etc.